### PR TITLE
updating Readme and adding arch doc

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
-# content-sources-backend
-Backend for content sources service
+# Content Sources
+
+##What is it?
+Content Sources is an application for storing content references (currently YUM repositories) in a central location.
```
Comment (rverdile) [inline README.md:4]: It might be good to define what a "content reference" is, maybe with an example.

File: docs/architecture.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Note: this document is targeting the desired architecture for the first milestone (November 2022).  It will be updated as needed. 
+
+This application consists of these parts:
+* Rest Api
+  * Used for our frontend or for other applications to interact with ours
+* Postgresql Database
+  * Used for storing our data
+* Kafka Platform
+  * Used to push messages for the event listener and other applications that want information around content sources
```
Comment (rverdile) [inline docs/architecture.md:13]: If there's an example you about what "information" would get pushed, that might be useful. But maybe it's too early for that :)

**Review by rverdile — COMMENTED** [review]
Just some suggestions. Feel free to disagree. Obviously I really like examples :).  I could see a small "example" section or doc, but maybe we don't have that information yet?

**Comment by jlsherrill** [comment]
good ideas @rverdile !    i reworded the first part and added examples for the rest.  Let me know if its clear or not

**Review by rverdile — APPROVED** [review]
Much clearer now! This looks good to me.

**Pull request merged**